index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header-container">
+        <h1 class="header-text">Welcome</h1>
+        <a href="#"><img src="./img/menu_icon.png" alt=""></a>
```

Comment:
- [ ] Good job adding the hamburger menu 👍🏼, but it should have a `nav` tag. The `nav` tag is for semantics reasons. Kindly, add `<nav></nav>` and within put the `<a>` tag. 🦕